src/fromager/commands/build.py

```diff
@@ -191,6 +209,11 @@ def _build(
         wheel_filename=wheel_filename,
     )
 
+    new_filename = wheel_log.with_name(wheel_filename.stem + ".log")
+    wheel_log.rename(new_filename)
+
+    per_wheel_logger.removeHandler(file_handler)
```

Comment:
We probably want to remove the handler and make sure it has closed the file before we rename it.

We should also think about the right thing to do if there's a build error. We won't upload the file in the production pipeline, but a developer debugging that issue locally is going to want to know how to find the log file and to know that it has everything in it.

Reply:
Hmm, okay. Any suggestions about what is the right thing to do? Add documentation for it?

Reply:
> We probably want to remove the handler and make sure it has closed the file before we rename it.

okay, done!